docs/zh_cn/notes/contribution_guides.md

```diff
@@ -0,0 +1,78 @@
+# 向MMSelfSup贡献代码
+
+- [向MMSelfSup贡献代码](#向MMSelfSup贡献代码)
+  - [流程](#流程)
+  - [代码风格](#代码风格)
+    - [Python](#python)
+    - [C++ and CUDA](#c-and-cuda)
+
+感谢您对参与MMSelfsup贡献的兴趣。我们欢迎各种形式的贡献，包括但不限于：
+
+- 修复 bug 或者错误
+- 添加或翻译文档为另外一种语言
+- 添加新特性或者组件
+
+## 流程
+
+我们推荐潜在的贡献者使用下面的流程来添加贡献：
+
+1. Fork 并 pull 最新的MMSelfSup仓库，并根据 [get_started](../get_started.md) 来配置环境。
+2. Checkout 一个新的分支 (\*\*不要使用 aster/dev branch 提交 PR)
+
+请从 `dev-1.x` 分支 checkout 一个新的分支，并按照下面指令：
+
+```bash
+git clone git@github.com:open-mmlab/mmselfsup.git
+cd mmselfsup
+git checkout dev-1.x
+git checkout -b xxxx # xxxx is the name of new branch
+```
+
+3. 根据后面提到的代码风格编辑相关的文件
+4. 使用 **pre-commit hook** 来检查和格式化您的修改。
+5. 提交您的修改。
+6. 创建一个 PR。
```

Comment:
```suggestion
6. 创建一个 PR，往 dev-1.x 合入。
```
Add this description of EN version simultaneously

Reply:
done